mergin/client.py

```diff
@@ -113,6 +116,21 @@ def __init__(self, url=None, auth_token=None, login=None, password=None, plugin_
             if not self._auth_session:
                 self.login(login, password)
 
+    def setup_logging(self):
+        """Setup Mergin client logging."""
+        client_log_file = os.environ.get('MERGIN_CLIENT_LOG', None)
+        self.log = logging.getLogger('mergin.client')
+        self.log.setLevel(logging.DEBUG)  # log everything (it would otherwise log just warnings+errors)
+        if not self.log.handlers:
+            if client_log_file:
+                log_handler = RotatingFileHandler(client_log_file, maxBytes=50*1024, backupCount=2)
```

Comment:
I would go with just ordinary `FileHandler` - we're not really interested in rotating as this could just add more files that will never be used...